src/common/histogram.h

```diff
@@ -29,7 +124,8 @@ struct pow2_hist_t { //
    * histogram
    *
    * bin size is 2^index
-   * value is count of elements that are <= the current bin but > the previous bin.
+   * value is count of elements that are <= the current bin but > the previous
+   * bin.
```

Comment:
Probably consequence of using a formatter -- don't include changes unrelated to your patch.